auth/src/main/java/com/firebase/ui/auth/util/signincontainer/IdpSignInContainer.java

```diff
@@ -69,7 +69,7 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
 
         if (providerConfig == null) {
             // we don't have a provider to handle this
-            finish(Activity.RESULT_CANCELED, new Intent());
+            finish(ResultCodes.UNKNOWN_ERROR, new Intent());
```

Comment:
Changing all of the CANCELED to UNKNOWN_ERROR is still a breaking API change, as people checking for CANCELED in code will no longer see the same behavior.

This makes me think maybe we should simplify to three cases:
  * OK
  * CANCELED (with optional extras describing the error)
  * NO_NETWORK (just to keep backwards compat, otherwise I'd say we lump this into CANCELED)